models/issues/label.go

```diff
@@ -194,11 +194,11 @@ func (label *Label) ExclusiveScope() string {
 	if !label.Exclusive {
 		return ""
 	}
-	lastIndex := strings.LastIndex(label.Name, "/")
-	if lastIndex == -1 {
+	index := strings.Index(label.Name, "/")
```

Comment:
Wait, why `Index` now and no longer `LastIndex`?
I am fine with both, but we need to clarify which one will be used before the merge…

Reply:
I'm not sure what I was thinking, I will put it back to last index since I did mean to have the scope include / even if we no longer do any special rendering for that.